release_notes/ocp-4-14-release-notes.adoc

```diff
@@ -363,6 +379,10 @@ For more information, see xref:../installing/install_config/installing-customizi
 With this release, control plane machine sets are supported for Nutanix clusters.
 
 For more information, see xref:../machine_management/control_plane_machine_management/cpmso-getting-started.adoc#cpmso-getting-started[Getting started with the Control Plane Machine Set Operator].
+[id="ocp-4-14-mco-ca-distribution"]
+==== Handling of registry certificate authorities
+
+The Machine Config Operator now handles distributing certificate authorities for image registries. This change does not affect end users.
```

Comment:
Is it important to include this in the release notes, in that case?

Reply:
This is not my change so unable to comment

Reply:
This is the JIRA requesting that it is documented https://issues.redhat.com/browse/OSDOCS-8026